src/Components/add/Add.tsx

```diff
@@ -5,15 +5,23 @@ import { useNavigate } from "react-router-dom";
 
 const URI = `https://localhost:7189/api/employees`;
 
+type Employee = {
```

Comment:
same as in c#, try to have single dedicated files for models and classes, etc. Create folder models and place the Employee in its own file

Reply:
better change in to class and create constructor with empty fields for easy use. Similar to 
![image](https://github.com/DDianovMD/ReactClientApp/assets/3842921/41a53bc2-3919-4cd5-9ee7-3b93ce22a8f3)